koku/masu/util/azure/common.py

```diff
@@ -179,7 +179,10 @@ def match_openshift_resources_and_labels(data_frame, cluster_topologies, matched
     volumes = chain.from_iterable(
         cluster_topology.get("persistent_volumes", []) for cluster_topology in cluster_topologies
     )
-    matchable_resources = list(nodes) + list(volumes)
+    csi_volume_handles = chain.from_iterable(
+        cluster_topology.get("csi_volume_handle", []) for cluster_topology in cluster_topologies
+    )
+    matchable_resources = list(nodes) + list(volumes) + list(csi_volume_handles)
```

Comment:
It's slightly more performant and probably uses less memory to use unpacking here.
```suggestion
    matchable_resources = [*nodes, *volumes, *csi_volume_handles]
```